### PR TITLE
Fixed bug in kmap map colors (issue #1558)

### PR DESCRIPTION
**Pull request by BFH-ktt1**
* #1558

### CODE REVIEW

File: src/main/java/com/cburch/logisim/analyze/data/CoverColor.java [hunk 1]
```diff
@@ -63,9 +63,8 @@ public Color getColor(int index) {
   }
 
   public Color getNext() {
-    index++;
     if (index < 0 || index >= colors.size()) index = 0;
```
Comment (MarcinOrlowski) [inline src/main/java/com/cburch/logisim/analyze/data/CoverColor.java:66]: Given the fact `index` is never decremented, the condition like `index < 0` makes no much sense. Can we have it removed (IIRC, two places in this file)?

Also CHANGELOG entry please.

**Review by MarcinOrlowski — CHANGES_REQUESTED** [review]

File: src/main/java/com/cburch/logisim/analyze/data/CoverColor.java (hunk 1 above)
Comment (BFH-ktt1) [reply src/main/java/com/cburch/logisim/analyze/data/CoverColor.java]: The first one was okay, as it was not addressing the local variable (I renamed it), the second one has been fixed and I added a changelog entry.

File: CHANGES.md [hunk 2]
```diff
@@ -5,6 +5,7 @@
 * @dev (????-??-??)
   * Added TTL 74182: look-ahead carry generator
   * Added TTL 74181: arithmetic logic unit
+  * Fixed Karnaughmap color index bug
```
Comment (maehne) [inline CHANGES.md:8]: "Karnaughmap" -> "Karnaugh map"

**Review by maehne — DISMISSED** [review]
Looks good to me except for one missing space `CHANGES.md`.

**Review by maehne — APPROVED** [review]

**Review by MarcinOrlowski — APPROVED** [review]

**Pull request merged**